starters/blog/src/components/bio.js

```diff
@@ -36,30 +36,34 @@ const Bio = () => {
   `)
 
   const { author, social } = data.site.siteMetadata
```

Comment:
How about adding comments, e.g. `// Set these values by editing siteMetadata in gatsby-config.js`